### PR TITLE
dev/translation#65 Further remove moneyValueFormat

### PR DESCRIPTION
**Pull request by eileenmcnaughton**
Overview
----------------------------------------
dev/translation#65 Further remove moneyValueFormat

Before
----------------------------------------
moneyValueFormat setting is used if set and deprecation notices are issues

After
----------------------------------------
Value no longer used

Technical Details
----------------------------------------
We deprecated this a while back. It is now removed except for behind the
scenes setting & getting of the value - which can go after we do
a universe search or after a few more months

@monishdeb @seamuslee001 @mattwire 

Comments
----------------------------------------
https://lab.civicrm.org/dev/translation/-/issues/65

### CODE REVIEW

File: api/v3/examples/Setting/GetFields.ex.php [hunk 1]
```diff
@@ -1774,21 +1774,6 @@ function setting_getfields_expectedresult() {
         'description' => '',
         'help_text' => '',
       ],
-      'moneyvalueformat' => [
```
Comment (seamuslee001) [inline api/v3/examples/Setting/GetFields.ex.php:1777]: I'm thinking this may cause a test failure but we will see

Comment (eileenmcnaughton) [reply api/v3/examples/Setting/GetFields.ex.php:1777]: It's in the example - I left the actual metadata

Comment (seamuslee001) [reply api/v3/examples/Setting/GetFields.ex.php:1777]: I have seen some unit tests refer to the examples as the expected output but we will see

**Comment by monishdeb** [comment]
@eileenmcnaughton  there are few other places which got the moneyValueFormat reference and would be safe to remove I guess. Like

```
CRM//Core/Config/MagicMerge.php:160
CRM//Utils/Check/Component/Schema.php:190
```
!!

**Comment by eileenmcnaughton** [comment]
@monishdeb yeah I wasn't sure about those as there is a possibility extensions access the setting (although not that likely)

**Comment by monishdeb** [comment]
Tested on local. Looks good to me

**Pull request merged**